backend/core/database.py

```diff
@@ -783,22 +786,25 @@ def handle_hijack_multiple_action(self, message):
                             entry[2]  # type
                         )
                         if action_is_related_to_seen:
-                            self.db_cur.execute(
-                                'UPDATE hijacks SET ' + action_ + ' WHERE key=%s;', (hijack_key, ))
+                            self.db_cur.execute(cmd_, (hijack_key, ))
                             self.db_conn.commit()
                         elif action_is_ignore:
-                            # if ongoing, force rekeying and delete persistent too
-                            if self.redis.sismember('persistent-keys', hijack_key):
-                                purge_redis_eph_pers_keys(self.redis, redis_hijack_key, hijack_key)
-                            self.db_cur.execute(
-                                'UPDATE hijacks SET ' + action_ + ' key=%s;', (hijack_key, ))
+                            # if ongoing, force rekeying and delete persistent
+                            # too
+                            if self.redis.sismember(
+                                    'persistent-keys', hijack_key):
+                                purge_redis_eph_pers_keys(
+                                    self.redis, redis_hijack_key, hijack_key)
+                            self.db_cur.execute(cmd_, (hijack_key, ))
                             self.db_conn.commit()
                         else:
```

Comment:
break this "else" in "elif" and "else", where "elif" will use the "action_is_resolve" flag (needs to be added above) and else will be the default no action (you can also put no "else" since we do nothing  in the unknown action case). With the resolve action, the timestamp is also taken into account as you have already done.

Reply:
resolve and ignore you mean